setup.py

```diff
@@ -83,18 +83,20 @@
     "filelock",
     "flake8>=3.8.3",
     "hf-doc-builder>=0.3.0",
-    "huggingface-hub>=0.8.1,<1.0",
+    "huggingface-hub>=0.8.1",
     "importlib_metadata",
     "isort>=5.5.4",
     "modelcards==0.1.4",
     "numpy",
     "pytest",
     "pytest-timeout",
     "pytest-xdist",
+    "scipy",
     "regex!=2019.12.17",
     "requests",
     "tensorboard",
     "torch>=1.4",
+    "transformers>=4.21.0",
```

Comment:
Need a relative recent version of Transformers so that this PR is included: https://github.com/huggingface/transformers/pull/18184